starknet-providers/src/sequencer_gateway.rs

```diff
@@ -57,6 +57,13 @@ impl SequencerGatewayProvider {
             Url::parse("https://alpha4.starknet.io/feeder_gateway").unwrap(),
         )
     }
+
+    pub fn starknet_alpha_localhost() -> Self {
```

Comment:
Can you rename the function to `starknet_nile_localhost`? I feel like it's more descriptive that way, as there are other local networks like the devnet.

Reply:
Good point! Pushed the change.